src/bp-activity/bp-activity-notifications.php

```diff
@@ -293,6 +293,45 @@ function bp_activity_remove_screen_notifications_single_activity_permalink( $act
 }
 add_action( 'bp_activity_screen_single_activity_permalink', 'bp_activity_remove_screen_notifications_single_activity_permalink' );
 
+/**
+ * Mark notification read when user visit sing post page permalink
+ *
+ * @since BuddyPress 2.0.0
```

Comment:
@janakkaneriya007 Please add the proper since tag name as now this is a platform not a BuddyPress and it should be like this `@since BuddyBoss 1.5.4`